include/git2/merge.h

```diff
@@ -181,8 +189,8 @@ typedef struct {
 	/** The file to favor in region conflicts. */
 	git_merge_file_favor_t favor;
 
-	/** see `git_merge_file_flags_t` above */
-	unsigned int flags;
+	/** see `git_merge_file_flag_t` above */
+	git_merge_file_flag_t flags;
```

Comment:
Multiple options from ```git_merge_file_flag_t``` may be specified, that is the diff style, whitespace options etc.